studio/stores/UiStore.ts

```diff
@@ -106,12 +107,16 @@ export default class UiStore implements IUiStore {
   }
 
   setProfile(value?: User) {
-    if (value?.id != this.profile?.id) {
+    if (value?.id !== this.profile?.id) {
       this.profile = value
 
       if (value) {
         Telemetry.sendIdentify(value)
       }
     }
   }
+
+  setProfileTotalFreeProjectsOwnedCount(value: number) {
+    this.profile = { ...this.profile, total_free_projects_owned: value } as User
+  }
```

Comment:
uiStore should not in control of total free project. Mutate useProfile and let swr gets the latest data instead

Reply:
gotcha - can i check real quick do i do something like this?
![image](https://user-images.githubusercontent.com/19742402/147065449-bbadb1af-cc1a-41e6-bdc1-8f9211054e0f.png)

Reply:
actually when should/do we use UiStore's profile and useProfile's profile?
I realised theres 2 places we're storing the user's profile and they aren't really in sync